src/test/kotlin/org/rust/lang/core/completion/RsAttributeCompletionTest.kt

```diff
@@ -268,4 +271,31 @@ class RsAttributeCompletionTest : RsAttributeCompletionTestBase() {
         #[non_/*caret*/]
         fn foo() {}
     """)
+
+    @ProjectDescriptor(WithProcMacroRustProjectDescriptor::class)
```

Comment:
Could you please rewrite tests using `WithDependencyRustProjectDescriptor` and `dep-proc-macro` package? Like we do it in `RsProcMacroResolveTest`, for example